Top-Level "Tools" section (#213)
buildpacks/docs: content/docs/tools/pack/cli/install.md

+++
title="Install"
weight=1
aliases=["/docs/install-pack/"]
summary="How to install the `pack` CLI using various methods."
+++

## Supported operating systems
You can install the most recent version of `pack` (version **{{< latest >}}**) as an executable binary on the following operating systems:

* [Linux](#linux)
* [macOS](#macos)
* [Windows](#windows)

---

# Linux

## Distro-Specific

#### Arch Linux

- [pack-cli](https://aur.archlinux.org/packages/pack-cli/)
- [pack-cli-bin](https://aur.archlinux.org/packages/pack-cli-bin/)

## Homebrew

```bash
brew install buildpacks/tap/pack
```

## Command

`pack` is available on GitHub releases so you can directly get and install it.

```bash
(curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.13.1/pack-v{{< latest >}}-linux.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
```

> **Optional:** Enable [auto-completion](#auto-completion)

---

# macOS

## Homebrew

To install `pack` on macOS and Linux, the easiest way is to use Homebrew:

```bash
brew install buildpacks/tap/pack
```

> **Optional:** Enable [auto-completion](#auto-completion)

---

## Windows

> To install `pack` on Windows, the easiest way is to use Chocolatey:

Reviewer: ```suggestion
To install `pack` on Windows, we recommend using Chocolatey:
```

---

> # Auto-completion

Reviewer 2: Now that pack completion works on both zsh and bash, perhaps we should just update this to reflect that?
